documentation/modules/ref-console-address-filtering.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// master.adoc
+
+[id='ref-console-address-filtering-{context}']
+= Using the {ProductName} Console address filtering
+
+The address name filtering feature in the {ProductName} Console uses regular expressions. Also, filters are cumulative.
+
+.{ProductName} Console address name filtering behavior
+[cols="34%a,33%a,33%a",options="header"]
+|===
+|To match... |Use this |Result
+|An expression exactly |Use a period followed by an expression: `.my` |The address `my`
```

Comment:
I'm not sure this is right. By 'exactly', do you mean match my but not amy or mya?

Reply:
@grs, yes, that is what I meant. I just tested it on my console, and it looks like to match "mya" you'd need to use `.my*` instead of `.my` ?

Reply:
I would expect 'my' to match 'mya' and 'amy' (anything with my in it). To match my and nothing else, I would use ^my$

Reply:
@grs, got it. Did some further testing and got that result. Making edits now